src/client/routes/dashboard/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect, Dispatch } from 'react-redux';
+import { IAppState } from '../../redux/reducers';
+import { actions } from './reducer';
+
+const Dashboard = () => {
+  return <div>Dashboard page</div>;
+};
+
+const mapStateToProps = (data: IAppState) => ({ data });
+
+// TODO - need to check whether any need to be remove or not
+const mapDispatchToProps = (dispatch: Dispatch<any>) => ({
+  actions: bindActionCreators(actions, dispatch)
```

Comment:
we should avoid pass actions like this, can cause readability issue

Reply:
little help would be required over here.